pkgs/by-name/cr/cryptsetup/package.nix

```diff
@@ -36,8 +36,10 @@ stdenv.mkDerivation rec {
   separateDebugInfo = true;
 
   src = fetchurl {
-    url = "mirror://kernel/linux/utils/cryptsetup/v${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
-    hash = "sha256-zJ4tN8JahxzqN1ILKNUyIHsMFnD7EPxU1oBx9j9SQ6I=";
+    url =
+      "mirror://kernel/linux/utils/cryptsetup/v${lib.versions.majorMinor finalAttrs.version}/"
+      + "${finalAttrs.pname}-${finalAttrs.version}.tar.xz";
```

Comment:
[`${pname}` should normally not be used in place of the package's name](https://github.com/NixOS/nixpkgs/issues/277994), we should simply hardcode `"cryptsetup"` here.